spec/helpers/application_helper_spec.rb

```diff
@@ -3,16 +3,19 @@
 require 'rails_helper'
 
 RSpec.describe ApplicationHelper, type: :helper do
-  describe '#derive_path' do
-    it 'calls the Rails path method that returns a route' do
-      expect(helper.derive_path('summaries')).to eq '/summaries'
+  describe '#nav_link_attributes' do
+    context 'when the current page matches the active controller' do
+      it 'produces user-friendly features indicating current location' do
+        expect(helper.nav_link_attributes('/test')).to eq({ aria: { current: 'page' }, class: 'nav-link '\
+                                                                'hover-change rounded-0 py-3 active-nav-link' })
+      end
     end
-  end
 
-  describe '#nav_link_attributes' do
-    it 'produces user-friendly features indicating current location' do
-      expect(helper.nav_link_attributes('test')).to eq({ aria: { current: 'page' }, class: 'nav-link hover-change'\
-                                                            ' rounded-0 py-3 active-nav-link' })
+    context 'when the current page does not match the active controller' do
```

Comment:
unrelated to work of patron controller, but I added this test because it was missing from earlier